docs/src/lib/representations.md

```diff
@@ -522,6 +522,7 @@ tovrep(::VPolytope)
 cartesian_product(::VPolytope{N}, ::VPolytope{N}) where N
 polyhedron(::VPolytope{N}) where {N<:Real}
 linear_map(::AbstractMatrix{N}, ::VPolytope{N}) where {N<:Real}
+minkowski_sum(::VPolytope{N},::VPolytope{N}) where {N<:Real}
```

Comment:
This function is in the wrong place (sorry if i told you to add it here): it should go under the `VPolygon` section of `representations.md`, because the new function acts on polygons, `minkowski_sum(::VPolygon{N}, ::VPolygon{N}) where {N<:Real}`.